arangod/MMFiles/MMFilesAqlFunctions.cpp

```diff
@@ -412,7 +412,7 @@ void MMFilesAqlFunctions::registerResources() {
   TRI_ASSERT(functions != nullptr);
 
   // fulltext functions
-  functions->add({"FULLTEXT", ".h,.,.|.", true, false,
+  functions->add({"FULLTEXT", ".h,.,.|.", false, false,
```

Comment:
but in this case the  FULLTEXT calculation node is not removed !??

Reply:
Yes, it is not in all cases. But I prefer stability or optimization in just some cases but crashes in others.